### ISSUE TITLE
Training Process

### ISSUE THREAD

**Issue by atosystem** [ElementAI/picard#26]
Hi @tscholak,

Can you provide your `loss`,  `eval_exact_match`, `eval_exec` for fine-tuning t5-3b and t5-base?

I would like to have a reference for my training process.

Thanks!

**Comment by tscholak** [comment]
Hi @atosystem, here is the loss and eval exact-match curves for a t5-3b job:
<img width="726" alt="Screen Shot 2021-11-13 at 2 41 24 PM" src="https://user-images.githubusercontent.com/1568873/141657032-0065ab17-60ee-497b-8265-a30bc894d74f.png">
the effective batch size for this run was 2048.

**Comment by atosystem** [comment]
Thanks @tscholak !
How about t5-base? Is the curve similar in shape but only different in the value?
About 0.57 in exact-match?

**Comment by tscholak** [comment]
Ok, here're the curves for [tscholak/1zha5ono](https://huggingface.co/tscholak/1zha5ono) which is based on t5.1.1.lm100k.base:
<img width="1121" alt="Screen Shot 2021-11-15 at 10 03 54 AM" src="https://user-images.githubusercontent.com/1568873/141804702-53fdb743-1669-42d8-8527-2d4fdf25f7f7.png">

**Comment by atosystem** [comment]
Thanks a lot!
